### PR TITLE
Allow JAE to pretrain on different datasets by storing hyperparams as json

### PR DESCRIPTION
**Pull request by rcannood**
@kimmo1019 Can you take a look at the proposed changes?

### CODE REVIEW

File: src/joint_embedding/methods/jae/train/script.py [hunk 1]
```diff
@@ -55,7 +56,7 @@
     mod1_reducer.fit(mod1_data)
```
Comment (kimmo1019) [inline src/joint_embedding/methods/jae/train/script.py:56]: In PCA transformation, the input should be raw count input_mod.layers['count'] rather than input_mod.X (please refer to the file 1_pca_pretrain.py I sent to you previously)
So the line 52, and line 64 here should be changed to 
`mod1_data = scale * normalize(input_mod1.layers["counts"],norm='l1', axis=1)`, mod2_data = scale * normalize(input_mod2.layers["counts"],norm='l1', axis=1)

**Review by kimmo1019 — COMMENTED** [review]
Storing params to json is awesome. 
Besides, I also noticed a small difference in PCA pretrain from the file I sent to you. Please see the comment there.

File: src/joint_embedding/methods/jae/train/script.py (hunk 1 above)
Comment (rcannood) [reply src/joint_embedding/methods/jae/train/script.py:56]: :+1: Good catch!

**Comment by rcannood** [comment]
My bad! Thanks for spotting this :)

**Comment by rcannood** [comment]
Woops.
